Cargo.lock

```diff
@@ -26727,7 +26737,7 @@ dependencies = [
  "array-bytes",
  "log",
  "parity-scale-codec",
- "polkadot-ckb-merkle-mountain-range",
+ "polkadot-ckb-merkle-mountain-range 0.8.1",
```

Comment:
why are there two versions of `sp-mmr-primitives` (and two versions of `polkadot-ckb-merkle-mountain-range`) used as deps? can you bump whatever is left behind?

Reply:
Looks like a "circular" dependency:

```
$ cargo update -p polkadot-ckb-merkle-mountain-range@0.7.0 --precise 0.8.1
    Updating crates.io index
error: failed to select a version for the requirement `polkadot-ckb-merkle-mountain-range = "^0.7.0"`
candidate versions found which didn't match: 0.8.1
location searched: crates.io index
required by package `sp-mmr-primitives v34.1.0`
    ... which satisfies dependency `sp-mmr-primitives = "^34.1.0"` (locked to 34.1.0) of package `polkadot-sdk v0.7.0`
    ... which satisfies dependency `polkadot-sdk = "^0.7"` (locked to 0.7.0) of package `subxt v0.38.0`
    ... which satisfies dependency `subxt = "^0.38"` (locked to 0.38.0) of package `polkadot-zombienet-sdk-tests v0.1.0 (/media/serban/data/workplace/sources/polkadot-sdk/polkadot/zombienet-sdk-tests)`
```

polkadot-zombienet-sdk-tests -> subxt = "^0.38" -> polkadot-sdk = "^0.7" -> sp-mmr-primitives = "^34.1.0" -> polkadot-ckb-merkle-mountain-range = "^0.7.0"

Reply:
can we fix the dep to `polkadot-zombienet-sdk-tests`? where are we using it?

Reply:
We are not using it anywhere, but the project is part of polkadot-sdk and it is listed under members in [`Cargo.toml`](https://github.com/paritytech/polkadot-sdk/blob/f4a196ab1473856c9c5992239fcc2f14c2c42914/Cargo.toml#L240) . I don't think we can remove it.

Reply:
Ok